lib/routes/steam/workshop-search.ts

```diff
@@ -0,0 +1,110 @@
+import { Route } from '@/types';
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { art } from '@/utils/render';
+import path from 'node:path';
+
+export const route: Route = {
+    path: '/workshopsearch/:appid/:routeParams?',
```

Comment:
```suggestion
    path: '/workshopsearch/:appid?/:routeParams?',
```
With `appid` having a default value in
https://github.com/DIYgod/RSSHub/blob/93928a8001c0dbd6410d1f83fbe48d81802800a7/lib/routes/steam/workshop-search.ts#L40
`appid` should be optional instead.